environment/src/main/java/com/sequenceiq/environment/proxy/domain/ProxyConfig.java

```diff
@@ -200,4 +200,24 @@ public boolean equals(Object o) {
     public int hashCode() {
         return Objects.hash(id);
     }
+
+    @Override
+    public String toString() {
+        return "ProxyConfig{" +
+                "id=" + id +
+                ", name='" + name + '\'' +
+                ", description='" + description + '\'' +
+                ", serverHost='" + serverHost + '\'' +
+                ", serverPort=" + serverPort +
+                ", protocol='" + protocol + '\'' +
+                ", userName=" + userName +
+                ", password=" + password +
```

Comment:
please skip password

Reply:
+1 And please explicitly check for the absence of the password in the UT.